chore: refactor package installation and handling during build time

inffy opened the pull request:
Refactor the package installation and COPR stuff

Closes https://github.com/ublue-os/aurora/issues/1115

inffy commented:
Well it builds now. Now just have to figure out if we are missing something important

renner0e commented:
> The patched fwupd...there was a specific reason and the patch was made by Kyle i think. But I cannot remember what the patch was about. There was discussion about it in discord just a while ago but i cannot find it

so the build option seems to hardcode the esp, I don't know why fedora doesn't need it but we do for some reason?

```
option(
  'efi_os_dir',
  type: 'string',
  description: 'the hardcoded name of OS directory in ESP, e.g. fedora',
```

EDIT: this is the reason, it will look in a bazzite directory, even tho it's called fedora

https://github.com/ublue-os/bazzite/issues/1400

renner0e commented:
any clue what happened on DX?

we seem to be missing incus, lxc ,kvmfr-kmod (we probably want to get rid of this one anyway) + some git and svn stuff got added

stable-dx -> this PR stable-dx

```
ostree diff commit from: booted deployment (48eb1b29e637185f52d17844ac1f4ae98775f20155845fb8585c05769a8f433d)
ostree diff commit to:   pending deployment (a49dcf070bda7b9f44ea8fcb74795b7cfd18ff5b737d6669a860cb5dbd5345f1)
Upgraded:
  fwupd 2.0.12-100.ublue -> 2.0.16-1.fc42
  fwupd-plugin-flashrom 2.0.12-100.ublue -> 2.0.16-1.fc42
  fwupd-plugin-modem-manager 2.0.12-100.ublue -> 2.0.16-1.fc42
  fwupd-plugin-uefi-capsule-data 2.0.12-100.ublue -> 2.0.16-1.fc42
  kcli 99.0.0.git.202510200604.f9a0558-0.fc42 -> 99.0.0.git.202510230957.8e499bf-0.fc42
  kernel-tools 6.16.8-200.fc42 -> 6.17.4-200.fc42
  kernel-tools-libs 6.16.8-200.fc42 -> 6.17.4-200.fc42
Downgraded:
  v4l2loopback 0.15.2-1.fc42 -> 0.15.1-1.fc42
Removed:
  add-determinism-0.6.0-1.fc42.x86_64
  adobe-source-code-pro-fonts-2.042.1.062.1.026-6.fc42.noarch
  akmod-v4l2loopback-0.15.2-1.fc42.x86_64
  akmods-0.6.2-1.fc42.noarch
  annobin-docs-12.94-1.fc42.noarch
  annobin-plugin-gcc-12.94-1.fc42.x86_64
  ansible-srpm-macros-1-17.1.fc42.noarch
  build-reproducibility-srpm-macros-0.6.0-1.fc42.noarch
  cascadia-code-fonts-2407.24-1.fc42.noarch
  cowsql-1.15.9-1.fc42.x86_64
  efi-srpm-macros-6-3.fc42.noarch
  fakeroot-1.37.1-1.fc42.x86_64
  fakeroot-libs-1.37.1-1.fc42.x86_64
  filesystem-srpm-macros-3.18-47.fc42.noarch
  firacode-nerd-fonts-3.4.0-1.fc42.noarch
  fonts-srpm-macros-1:2.0.5-22.fc42.noarch
  forge-srpm-macros-0.4.0-2.fc42.noarch
  fpc-srpm-macros-1.3-14.fc42.noarch
  gcc-plugin-annobin-15.2.1-1.fc42.x86_64
  ghc-srpm-macros-1.9.2-2.fc42.noarch
  gnat-srpm-macros-6-7.fc42.noarch
  go-srpm-macros-3.8.0-1.fc42.noarch
  google-droid-sans-mono-fonts-20200215-22.fc42.noarch
  google-go-mono-fonts-0.23.0-2.fc42.noarch
  grubby-8.40-82.fc42.x86_64
  ibm-plex-mono-fonts-6.4.0-3.fc42.noarch
  incus-6.15-1.fc42.x86_64
  incus-agent-6.15-1.fc42.x86_64
  incus-client-6.15-1.fc42.x86_64
  incus-selinux-6.15-1.fc42.noarch
  intel-one-mono-fonts-1.4.0-1.fc42.noarch
  jetbrains-mono-fonts-2.304-8.fc42.noarch
  jetbrains-mono-fonts-all-2.304-8.fc42.noarch
  jetbrains-mono-nl-fonts-2.304-8.fc42.noarch
  kernel-srpm-macros-1.0-25.fc42.noarch
  kmod-kvmfr-0.0.git.32.e2d4889e-1.fc42.x86_64
  kmodtool-1.1-13.fc42.noarch
  kvmfr-0.0.git.32.e2d4889e-1.fc42.x86_64
  lua-srpm-macros-1-15.fc42.noarch
  lxc-6.0.5-1.fc42.x86_64
  lxc-libs-6.0.5-1.fc42.x86_64
  lxc-templates-6.0.5-1.fc42.x86_64
  lxcfs-6.0.5-1.fc42.x86_64
  mozilla-fira-fonts-common-4.202-21.fc42.noarch
  mozilla-fira-mono-fonts-4.202-21.fc42.noarch
  nerdfontssymbolsonly-nerd-fonts-3.4.0-1.fc42.noarch
  ocaml-srpm-macros-10-4.fc42.noarch
  openblas-srpm-macros-2-19.fc42.noarch
  package-notes-srpm-macros-0.5-13.fc42.noarch
  perl-srpm-macros-1-57.fc42.noarch
  pyproject-srpm-macros-1.18.4-1.fc42.noarch
  python-srpm-macros-3.13-5.fc42.noarch
  python3-progressbar2-4.5.0-1.fc42.noarch
  python3-rpmautospec-0.8.1-11.fc42.noarch
  python3-rpmautospec-core-0.1.5-3.fc42.noarch
  python3-utils-3.9.0-2.fc42.noarch
  qt5-srpm-macros-5.15.17-1.fc42.noarch
  qt6-srpm-macros-6.9.2-1.fc42.noarch
  raft-0.22.1-3.fc42.x86_64
  redhat-rpm-config-342-4.fc42.noarch
  rpm-build-4.20.1-1.fc42.x86_64
  rpmdevtools-9.6-9.fc42.noarch
  rsms-inter-fonts-4.1-1.fc42.noarch
  rust-srpm-macros-26.4-1.fc42.noarch
  starship-1.23.0-1.fc42.x86_64
  systemd-rpm-macros-257.10-1.fc42.noarch
  tree-sitter-srpm-macros-0.1.0-8.fc42.noarch
  xdelta-3.1.0-22.fc42.x86_64
  zig-srpm-macros-1-4.fc42.noarch
Added:
  git-subtree-2.51.0-2.fc42.noarch
  git-svn-2.51.0-2.fc42.noarch
  libretls-3.8.1-5.fc42.x86_64
  libserf-1.3.10-9.fc42.x86_64
  netcat-1.229-2.fc42.x86_64
  perl-Git-SVN-2.51.0-2.fc42.noarch
  perl-Memoize-1.16-519.fc42.noarch
  perl-YAML-1.31-5.fc42.noarch
  subversion-1.14.5-14.fc42.x86_64
  subversion-libs-1.14.5-14.fc42.x86_64
  subversion-perl-1.14.5-14.fc42.x86_64
  utf8proc-2.10.0-1.fc42.x86_64
```

inffy commented:
lxc and incus seem to have packages in fedora now and they were missing from the package list.

should we remove those git/svn stuff?

```
 git-subtree
    git-svn
```
these two i think

inffy commented:
aurora-dx:stable (main) -> aurora-dx:latest (this pr branch)

```
ostree diff commit from: booted deployment (63f3cadc0061ef030fc9ae11b65bcd3e10b2f20cded8f0c646fb5a6e080ac8f2)
ostree diff commit to:   pending deployment (35359afa06b7e6c232bb56470ba3dfe6c507ef9b07c64673201e85e605c4492a)
Upgraded:
  Sunshine 2025.1014.193231-1.fc42.x86_64 -> 2025.1021.185115-1.fc42.x86_64
  fwupd 2.0.12-100.ublue.x86_64 -> 2.0.16-1.fc42.x86_64
  fwupd-plugin-flashrom 2.0.12-100.ublue.x86_64 -> 2.0.16-1.fc42.x86_64
  fwupd-plugin-modem-manager 2.0.12-100.ublue.x86_64 -> 2.0.16-1.fc42.x86_64
  fwupd-plugin-uefi-capsule-data 2.0.12-100.ublue.x86_64 -> 2.0.16-1.fc42.x86_64
    * Fri Sep 12 2025 Richard Hughes <richard@hughsie.com> - 2.0.16-1
    - New upstream release

    * Wed Sep 10 2025 Richard Hughes <richard@hughsie.com> - 2.0.15-1
    - New upstream release

    * Fri Aug 29 2025 Richard Hughes <richard@hughsie.com> - 2.0.14-1
    - New upstream release

    * Tue Jul 22 2025 Richard Hughes <richard@hughsie.com> - 2.0.13-1
    - New upstream release

  ibus-typing-booster 2.28.0-1.fc42.noarch -> 2.28.1-1.fc42.noarch
    * Thu Oct 16 2025 Mike FABIAN <mfabian@redhat.com> - 2.28.1-1
    - Update to 2.28.1
    - Ignore “Enable suggestions by key” and “Minimum number of chars for
      completion“ options when OSK is used Resolves: https://github.com/mike-
      fabian/ibus-typing-booster/issues/813 Resolves: https://github.com/mike-
      fabian/ibus-typing-booster/issues/814
    - Translation update from Weblate (es 100%, tr 100%, zh_CN 73.1%)

  kernel 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
    * Sun Oct 19 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.4-0]
    - Add Fedora 42 and 41 to release_targets (Justin M. Forbes)
    - RHEL_RELEASE should be 0, not 1 (Justin M. Forbes)
    - PCI: vmd: override irq_startup()/irq_shutdown() in vmd_init_dev_msi_info() (Inochi Amaoto)
    - fedora: arm64: Updates for AMD Xilinx devices (Peter Robinson)
    - Fix up config issue due to stable backport (Justin M. Forbes)
    - Linux v6.17.4

    * Wed Oct 15 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.3-1]
    - Linux v6.17.3

    * Sun Oct 12 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.2-1]
    - fedora: aarch64: Enable arm MHUv2 driver (Peter Robinson)
    - fedora: arm: Enable the NVMEM_IMX_OCOTP_ELE module (Peter Robinson)
    - Linux v6.17.2

    * Mon Oct 06 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.1-1]
    - Add Bug to BugsFixed (Justin M. Forbes)
    - gpio: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - i2c: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - wifi: ath11k: Add missing platform IDs for quirk table (Mark Pearson)
    - blk-mq: fix blk_mq_tags double free while nr_requests grown (Yu Kuai)
    - usb: typec: ucsi: Handle incorrect num_connectors capability (Mark Pearson)
    - Initial setup for stable Fedora releases (Justin M. Forbes)
    - arm64: dts: qcom: x1e80100-lenovo-yoga-slim7x: add Bluetooth support (Jens Glathe)
    - redhat: config: Enable USBIO modules (Hans de Goede)
    - i2c: Add Intel USBIO I2C driver (Israel Cepeda)
    - gpio: Add Intel USBIO GPIO driver (Israel Cepeda)
    - usb: misc: Add Intel USBIO bridge driver (Israel Cepeda)
    - media: ov08x40: Fix the horizontal flip control (Hao Yao)
    - ACPI: scan: Add Intel CVS ACPI HIDs to acpi_ignore_dep_ids[] (Hans de Goede)
    - platform/x86: int3472: Increase ov08x40 handshake GPIO delay to 45 ms (Hans de Goede)
    - platform/x86: int3472: Rework regulator enable-time handling (Hans de Goede)
    - platform/x86: int3472: Convert int3472_gpio_map to use C99 initializers (Hans de Goede)
    - powerpc/tools: drop `-o pipefail` in gcc check scripts (Jan Stancek)
    - redhat/configs: enable CONFIG_KVM_INTEL_TDX for Fedora x86 (Daniel P. Berrangé)
    - KVM/TDX: Explicitly do WBINVD when no more TDX SEAMCALLs (Kai Huang)
    - x86/virt/tdx: Update the kexec section in the TDX documentation (Kai Huang)
    - x86/virt/tdx: Remove the !KEXEC_CORE dependency (Kai Huang)
    - x86/kexec: Disable kexec/kdump on platforms with TDX partial write erratum (Kai Huang)
    - x86/virt/tdx: Mark memory cache state incoherent when making SEAMCALL (Kai Huang)
    - x86/sme: Use percpu boolean to control WBINVD during kexec (Kai Huang)
    - x86/kexec: Consolidate relocate_kernel() function parameters (Kai Huang)
    - Linux v6.17.1

    * Mon Sep 29 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.0-1]
    - Reset RHEL_RELEASE for the 6.18 cycle (Justin M. Forbes)
    - Turn on USB_FUNCTIONFS for Fedora (Justin M. Forbes)
    - redhat/configs: Disable CONFIG_EFI_MIXED in RHEL (Lenny Szubowicz)

    * Mon Sep 29 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-62]
    - Linux v6.17.0

    * Sun Sep 28 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.51a24b7deaae.61]
    - Linux v6.17.0-0.rc7.51a24b7deaae

    * Sat Sep 27 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.fec734e8d564.60]
    - Revert "Merge branch 'tmp2' into 'os-build'" (Justin M. Forbes)
    - Linux v6.17.0-0.rc7.fec734e8d564

    * Fri Sep 26 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.4ff71af020ae.59]
    - Always set LLVM=1 when building with clang (Tom Stellard)
    - redhat/configs: Move CONFIG_MITIGATION_VMSCAPE to common/generic/x86 (Waiman Long)
    - Linux v6.17.0-0.rc7.4ff71af020ae

    * Thu Sep 25 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.bf40f4b87761.58]
    - redhat/Makefile: Update the make target dist-configs-check to fail (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.bf40f4b87761

    * Wed Sep 24 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.57]
    - Consolidate configs to common for 6.17 (Justin M. Forbes)

    * Tue Sep 23 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.56]
    - Add 1010-config-newlines-test.bats self test. (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.cec1e6e5d1ab

    * Mon Sep 22 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.55]
    - Linux v6.17.0-0.rc7

    * Sun Sep 21 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.f975f08c2e89.54]
    - Linux v6.17.0-0.rc6.f975f08c2e89

    * Sat Sep 20 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.cd89d487374c.53]
    - Linux v6.17.0-0.rc6.cd89d487374c

    * Fri Sep 19 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.097a6c336d00.52]
    - os-build: Remove dead CONFIG_SCHED_DEBUG files (Phil Auld)
    - redhat/configs: automotive: Disable COMPAT_32BIT_TIME SGETMASK_SYSCALL and IA32_EMULATION configs (Dorinda Bassey)
    - Linux v6.17.0-0.rc6.097a6c336d00

  kernel-core 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
    * Sun Oct 19 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.4-0]
    - Add Fedora 42 and 41 to release_targets (Justin M. Forbes)
    - RHEL_RELEASE should be 0, not 1 (Justin M. Forbes)
    - PCI: vmd: override irq_startup()/irq_shutdown() in vmd_init_dev_msi_info() (Inochi Amaoto)
    - fedora: arm64: Updates for AMD Xilinx devices (Peter Robinson)
    - Fix up config issue due to stable backport (Justin M. Forbes)
    - Linux v6.17.4

    * Wed Oct 15 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.3-1]
    - Linux v6.17.3

    * Sun Oct 12 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.2-1]
    - fedora: aarch64: Enable arm MHUv2 driver (Peter Robinson)
    - fedora: arm: Enable the NVMEM_IMX_OCOTP_ELE module (Peter Robinson)
    - Linux v6.17.2

    * Mon Oct 06 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.1-1]
    - Add Bug to BugsFixed (Justin M. Forbes)
    - gpio: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - i2c: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - wifi: ath11k: Add missing platform IDs for quirk table (Mark Pearson)
    - blk-mq: fix blk_mq_tags double free while nr_requests grown (Yu Kuai)
    - usb: typec: ucsi: Handle incorrect num_connectors capability (Mark Pearson)
    - Initial setup for stable Fedora releases (Justin M. Forbes)
    - arm64: dts: qcom: x1e80100-lenovo-yoga-slim7x: add Bluetooth support (Jens Glathe)
    - redhat: config: Enable USBIO modules (Hans de Goede)
    - i2c: Add Intel USBIO I2C driver (Israel Cepeda)
    - gpio: Add Intel USBIO GPIO driver (Israel Cepeda)
    - usb: misc: Add Intel USBIO bridge driver (Israel Cepeda)
    - media: ov08x40: Fix the horizontal flip control (Hao Yao)
    - ACPI: scan: Add Intel CVS ACPI HIDs to acpi_ignore_dep_ids[] (Hans de Goede)
    - platform/x86: int3472: Increase ov08x40 handshake GPIO delay to 45 ms (Hans de Goede)
    - platform/x86: int3472: Rework regulator enable-time handling (Hans de Goede)
    - platform/x86: int3472: Convert int3472_gpio_map to use C99 initializers (Hans de Goede)
    - powerpc/tools: drop `-o pipefail` in gcc check scripts (Jan Stancek)
    - redhat/configs: enable CONFIG_KVM_INTEL_TDX for Fedora x86 (Daniel P. Berrangé)
    - KVM/TDX: Explicitly do WBINVD when no more TDX SEAMCALLs (Kai Huang)
    - x86/virt/tdx: Update the kexec section in the TDX documentation (Kai Huang)
    - x86/virt/tdx: Remove the !KEXEC_CORE dependency (Kai Huang)
    - x86/kexec: Disable kexec/kdump on platforms with TDX partial write erratum (Kai Huang)
    - x86/virt/tdx: Mark memory cache state incoherent when making SEAMCALL (Kai Huang)
    - x86/sme: Use percpu boolean to control WBINVD during kexec (Kai Huang)
    - x86/kexec: Consolidate relocate_kernel() function parameters (Kai Huang)
    - Linux v6.17.1

    * Mon Sep 29 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.0-1]
    - Reset RHEL_RELEASE for the 6.18 cycle (Justin M. Forbes)
    - Turn on USB_FUNCTIONFS for Fedora (Justin M. Forbes)
    - redhat/configs: Disable CONFIG_EFI_MIXED in RHEL (Lenny Szubowicz)

    * Mon Sep 29 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-62]
    - Linux v6.17.0

    * Sun Sep 28 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.51a24b7deaae.61]
    - Linux v6.17.0-0.rc7.51a24b7deaae

    * Sat Sep 27 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.fec734e8d564.60]
    - Revert "Merge branch 'tmp2' into 'os-build'" (Justin M. Forbes)
    - Linux v6.17.0-0.rc7.fec734e8d564

    * Fri Sep 26 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.4ff71af020ae.59]
    - Always set LLVM=1 when building with clang (Tom Stellard)
    - redhat/configs: Move CONFIG_MITIGATION_VMSCAPE to common/generic/x86 (Waiman Long)
    - Linux v6.17.0-0.rc7.4ff71af020ae

    * Thu Sep 25 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.bf40f4b87761.58]
    - redhat/Makefile: Update the make target dist-configs-check to fail (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.bf40f4b87761

    * Wed Sep 24 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.57]
    - Consolidate configs to common for 6.17 (Justin M. Forbes)

    * Tue Sep 23 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.56]
    - Add 1010-config-newlines-test.bats self test. (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.cec1e6e5d1ab

    * Mon Sep 22 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.55]
    - Linux v6.17.0-0.rc7

    * Sun Sep 21 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.f975f08c2e89.54]
    - Linux v6.17.0-0.rc6.f975f08c2e89

    * Sat Sep 20 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.cd89d487374c.53]
    - Linux v6.17.0-0.rc6.cd89d487374c

    * Fri Sep 19 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.097a6c336d00.52]
    - os-build: Remove dead CONFIG_SCHED_DEBUG files (Phil Auld)
    - redhat/configs: automotive: Disable COMPAT_32BIT_TIME SGETMASK_SYSCALL and IA32_EMULATION configs (Dorinda Bassey)
    - Linux v6.17.0-0.rc6.097a6c336d00

  kernel-devel 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
  kernel-devel-matched 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
    * Sun Oct 19 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.4-0]
    - Add Fedora 42 and 41 to release_targets (Justin M. Forbes)
    - RHEL_RELEASE should be 0, not 1 (Justin M. Forbes)
    - PCI: vmd: override irq_startup()/irq_shutdown() in vmd_init_dev_msi_info() (Inochi Amaoto)
    - fedora: arm64: Updates for AMD Xilinx devices (Peter Robinson)
    - Fix up config issue due to stable backport (Justin M. Forbes)
    - Linux v6.17.4

    * Wed Oct 15 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.3-1]
    - Linux v6.17.3

    * Sun Oct 12 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.2-1]
    - fedora: aarch64: Enable arm MHUv2 driver (Peter Robinson)
    - fedora: arm: Enable the NVMEM_IMX_OCOTP_ELE module (Peter Robinson)
    - Linux v6.17.2

    * Mon Oct 06 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.1-1]
    - Add Bug to BugsFixed (Justin M. Forbes)
    - gpio: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - i2c: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - wifi: ath11k: Add missing platform IDs for quirk table (Mark Pearson)
    - blk-mq: fix blk_mq_tags double free while nr_requests grown (Yu Kuai)
    - usb: typec: ucsi: Handle incorrect num_connectors capability (Mark Pearson)
    - Initial setup for stable Fedora releases (Justin M. Forbes)
    - arm64: dts: qcom: x1e80100-lenovo-yoga-slim7x: add Bluetooth support (Jens Glathe)
    - redhat: config: Enable USBIO modules (Hans de Goede)
    - i2c: Add Intel USBIO I2C driver (Israel Cepeda)
    - gpio: Add Intel USBIO GPIO driver (Israel Cepeda)
    - usb: misc: Add Intel USBIO bridge driver (Israel Cepeda)
    - media: ov08x40: Fix the horizontal flip control (Hao Yao)
    - ACPI: scan: Add Intel CVS ACPI HIDs to acpi_ignore_dep_ids[] (Hans de Goede)
    - platform/x86: int3472: Increase ov08x40 handshake GPIO delay to 45 ms (Hans de Goede)
    - platform/x86: int3472: Rework regulator enable-time handling (Hans de Goede)
    - platform/x86: int3472: Convert int3472_gpio_map to use C99 initializers (Hans de Goede)
    - powerpc/tools: drop `-o pipefail` in gcc check scripts (Jan Stancek)
    - redhat/configs: enable CONFIG_KVM_INTEL_TDX for Fedora x86 (Daniel P. Berrangé)
    - KVM/TDX: Explicitly do WBINVD when no more TDX SEAMCALLs (Kai Huang)
    - x86/virt/tdx: Update the kexec section in the TDX documentation (Kai Huang)
    - x86/virt/tdx: Remove the !KEXEC_CORE dependency (Kai Huang)
    - x86/kexec: Disable kexec/kdump on platforms with TDX partial write erratum (Kai Huang)
    - x86/virt/tdx: Mark memory cache state incoherent when making SEAMCALL (Kai Huang)
    - x86/sme: Use percpu boolean to control WBINVD during kexec (Kai Huang)
    - x86/kexec: Consolidate relocate_kernel() function parameters (Kai Huang)
    - Linux v6.17.1

    * Mon Sep 29 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.0-1]
    - Reset RHEL_RELEASE for the 6.18 cycle (Justin M. Forbes)
    - Turn on USB_FUNCTIONFS for Fedora (Justin M. Forbes)
    - redhat/configs: Disable CONFIG_EFI_MIXED in RHEL (Lenny Szubowicz)

    * Mon Sep 29 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-62]
    - Linux v6.17.0

    * Sun Sep 28 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.51a24b7deaae.61]
    - Linux v6.17.0-0.rc7.51a24b7deaae

    * Sat Sep 27 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.fec734e8d564.60]
    - Revert "Merge branch 'tmp2' into 'os-build'" (Justin M. Forbes)
    - Linux v6.17.0-0.rc7.fec734e8d564

    * Fri Sep 26 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.4ff71af020ae.59]
    - Always set LLVM=1 when building with clang (Tom Stellard)
    - redhat/configs: Move CONFIG_MITIGATION_VMSCAPE to common/generic/x86 (Waiman Long)
    - Linux v6.17.0-0.rc7.4ff71af020ae

    * Thu Sep 25 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.bf40f4b87761.58]
    - redhat/Makefile: Update the make target dist-configs-check to fail (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.bf40f4b87761

    * Wed Sep 24 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.57]
    - Consolidate configs to common for 6.17 (Justin M. Forbes)

    * Tue Sep 23 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.56]
    - Add 1010-config-newlines-test.bats self test. (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.cec1e6e5d1ab

    * Mon Sep 22 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.55]
    - Linux v6.17.0-0.rc7

    * Sun Sep 21 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.f975f08c2e89.54]
    - Linux v6.17.0-0.rc6.f975f08c2e89

    * Sat Sep 20 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.cd89d487374c.53]
    - Linux v6.17.0-0.rc6.cd89d487374c

    * Fri Sep 19 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.097a6c336d00.52]
    - os-build: Remove dead CONFIG_SCHED_DEBUG files (Phil Auld)
    - redhat/configs: automotive: Disable COMPAT_32BIT_TIME SGETMASK_SYSCALL and IA32_EMULATION configs (Dorinda Bassey)
    - Linux v6.17.0-0.rc6.097a6c336d00

  kernel-headers 6.16.2-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
    * Sun Oct 19 2025 Justin M. Forbes <jforbes@fedoraproject.org> - 6.17.4-1
    - Linux v6.17.4

  kernel-modules 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
  kernel-modules-core 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
  kernel-modules-extra 6.16.8-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
    * Sun Oct 19 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.4-0]
    - Add Fedora 42 and 41 to release_targets (Justin M. Forbes)
    - RHEL_RELEASE should be 0, not 1 (Justin M. Forbes)
    - PCI: vmd: override irq_startup()/irq_shutdown() in vmd_init_dev_msi_info() (Inochi Amaoto)
    - fedora: arm64: Updates for AMD Xilinx devices (Peter Robinson)
    - Fix up config issue due to stable backport (Justin M. Forbes)
    - Linux v6.17.4

    * Wed Oct 15 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.3-1]
    - Linux v6.17.3

    * Sun Oct 12 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.2-1]
    - fedora: aarch64: Enable arm MHUv2 driver (Peter Robinson)
    - fedora: arm: Enable the NVMEM_IMX_OCOTP_ELE module (Peter Robinson)
    - Linux v6.17.2

    * Mon Oct 06 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.1-1]
    - Add Bug to BugsFixed (Justin M. Forbes)
    - gpio: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - i2c: usbio: Add ACPI device-id for MTL-CVF devices (Hans de Goede)
    - wifi: ath11k: Add missing platform IDs for quirk table (Mark Pearson)
    - blk-mq: fix blk_mq_tags double free while nr_requests grown (Yu Kuai)
    - usb: typec: ucsi: Handle incorrect num_connectors capability (Mark Pearson)
    - Initial setup for stable Fedora releases (Justin M. Forbes)
    - arm64: dts: qcom: x1e80100-lenovo-yoga-slim7x: add Bluetooth support (Jens Glathe)
    - redhat: config: Enable USBIO modules (Hans de Goede)
    - i2c: Add Intel USBIO I2C driver (Israel Cepeda)
    - gpio: Add Intel USBIO GPIO driver (Israel Cepeda)
    - usb: misc: Add Intel USBIO bridge driver (Israel Cepeda)
    - media: ov08x40: Fix the horizontal flip control (Hao Yao)
    - ACPI: scan: Add Intel CVS ACPI HIDs to acpi_ignore_dep_ids[] (Hans de Goede)
    - platform/x86: int3472: Increase ov08x40 handshake GPIO delay to 45 ms (Hans de Goede)
    - platform/x86: int3472: Rework regulator enable-time handling (Hans de Goede)
    - platform/x86: int3472: Convert int3472_gpio_map to use C99 initializers (Hans de Goede)
    - powerpc/tools: drop `-o pipefail` in gcc check scripts (Jan Stancek)
    - redhat/configs: enable CONFIG_KVM_INTEL_TDX for Fedora x86 (Daniel P. Berrangé)
    - KVM/TDX: Explicitly do WBINVD when no more TDX SEAMCALLs (Kai Huang)
    - x86/virt/tdx: Update the kexec section in the TDX documentation (Kai Huang)
    - x86/virt/tdx: Remove the !KEXEC_CORE dependency (Kai Huang)
    - x86/kexec: Disable kexec/kdump on platforms with TDX partial write erratum (Kai Huang)
    - x86/virt/tdx: Mark memory cache state incoherent when making SEAMCALL (Kai Huang)
    - x86/sme: Use percpu boolean to control WBINVD during kexec (Kai Huang)
    - x86/kexec: Consolidate relocate_kernel() function parameters (Kai Huang)
    - Linux v6.17.1

    * Mon Sep 29 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.0-1]
    - Reset RHEL_RELEASE for the 6.18 cycle (Justin M. Forbes)
    - Turn on USB_FUNCTIONFS for Fedora (Justin M. Forbes)
    - redhat/configs: Disable CONFIG_EFI_MIXED in RHEL (Lenny Szubowicz)

    * Mon Sep 29 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-62]
    - Linux v6.17.0

    * Sun Sep 28 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.51a24b7deaae.61]
    - Linux v6.17.0-0.rc7.51a24b7deaae

    * Sat Sep 27 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.fec734e8d564.60]
    - Revert "Merge branch 'tmp2' into 'os-build'" (Justin M. Forbes)
    - Linux v6.17.0-0.rc7.fec734e8d564

    * Fri Sep 26 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.4ff71af020ae.59]
    - Always set LLVM=1 when building with clang (Tom Stellard)
    - redhat/configs: Move CONFIG_MITIGATION_VMSCAPE to common/generic/x86 (Waiman Long)
    - Linux v6.17.0-0.rc7.4ff71af020ae

    * Thu Sep 25 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.bf40f4b87761.58]
    - redhat/Makefile: Update the make target dist-configs-check to fail (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.bf40f4b87761

    * Wed Sep 24 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.57]
    - Consolidate configs to common for 6.17 (Justin M. Forbes)

    * Tue Sep 23 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.cec1e6e5d1ab.56]
    - Add 1010-config-newlines-test.bats self test. (Alexandra Hájková)
    - Linux v6.17.0-0.rc7.cec1e6e5d1ab

    * Mon Sep 22 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc7.55]
    - Linux v6.17.0-0.rc7

    * Sun Sep 21 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.f975f08c2e89.54]
    - Linux v6.17.0-0.rc6.f975f08c2e89

    * Sat Sep 20 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.cd89d487374c.53]
    - Linux v6.17.0-0.rc6.cd89d487374c

    * Fri Sep 19 2025 Fedora Kernel Team <kernel-team@fedoraproject.org> [6.17.0-0.rc6.097a6c336d00.52]
    - os-build: Remove dead CONFIG_SCHED_DEBUG files (Phil Auld)
    - redhat/configs: automotive: Disable COMPAT_32BIT_TIME SGETMASK_SYSCALL and IA32_EMULATION configs (Dorinda Bassey)
    - Linux v6.17.0-0.rc6.097a6c336d00

  kernel-tools 6.16.12-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
  kernel-tools-libs 6.16.12-200.fc42.x86_64 -> 6.17.4-200.fc42.x86_64
    * Sun Oct 19 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.4-0]
    - Add Fedora 42 and 41 to release_targets (Justin M. Forbes)
    - RHEL_RELEASE should be 0, not 1 (Justin M. Forbes)
    - PCI: vmd: override irq_startup()/irq_shutdown() in vmd_init_dev_msi_info() (Inochi Amaoto)
    - fedora: arm64: Updates for AMD Xilinx devices (Peter Robinson)
    - Fix up config issue due to stable backport (Justin M. Forbes)
    - Linux v6.17.4

    * Wed Oct 15 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.3-1]
    - Linux v6.17.3

    * Sun Oct 12 2025 Justin M. Forbes <jforbes@fedoraproject.org> [6.17.2-1]
    - fedora: aarch64: Enable arm MHUv2 driver (Peter Robinson)
    - fedora: arm: Enable the NVMEM_IMX_OCOTP_ELE module (Peter Robinson)
    - Linux v6.17.2

  mbedtls 3.6.4-1.fc42.x86_64 -> 3.6.5-1.fc42.x86_64
    * Thu Oct 16 2025 Morten Stevens <mstevens@fedoraproject.org> - 3.6.5-1
    - Update to 3.6.5

    * Thu Jul 24 2025 Fedora Release Engineering <releng@fedoraproject.org> - 3.6.4-3
    - Rebuilt for https://fedoraproject.org/wiki/Fedora_43_Mass_Rebuild

    * Thu Jul 03 2025 Morten Stevens <mstevens@fedoraproject.org> - 3.6.4-2
    - Add conditional CFLAGS for Fedora and RHEL

  python3-boto3 1.40.51-1.fc42.noarch -> 1.40.54-1.fc42.noarch
    * Thu Oct 16 2025 Gwyn Ciesla <gwync@protonmail.com> - 1.40.54-1
    - 1.40.54

    * Wed Oct 15 2025 Gwyn Ciesla <gwync@protonmail.com> - 1.40.53-1
    - 1.40.53

    * Wed Oct 15 2025 Gwyn Ciesla <gwync@protonmail.com> - 1.40.52-1
    - 1.40.52

  python3-botocore 1.40.51-1.fc42.noarch -> 1.40.54-1.fc42.noarch
    * Thu Oct 16 2025 Gwyn Ciesla <gwync@protonmail.com> - 1.40.54-1
    - 1.40.54

    * Wed Oct 15 2025 Gwyn Ciesla <gwync@protonmail.com> - 1.40.53-1
    - 1.40.53

    * Wed Oct 15 2025 Gwyn Ciesla <gwync@protonmail.com> - 1.40.52-1
    - 1.40.52

  tailscale 1.88.4-1.x86_64 -> 1.90.1-1.x86_64
Downgraded:
  v4l2loopback 0.15.2-1.fc42.x86_64 -> 0.15.1-1.fc42.x86_64
Removed:
  add-determinism-0.6.0-1.fc42.x86_64
  akmod-v4l2loopback-0.15.2-1.fc42.x86_64
  akmods-0.6.2-1.fc42.noarch
  annobin-docs-12.94-1.fc42.noarch
  annobin-plugin-gcc-12.94-1.fc42.x86_64
  ansible-srpm-macros-1-17.1.fc42.noarch
  build-reproducibility-srpm-macros-0.6.0-1.fc42.noarch
  efi-srpm-macros-6-3.fc42.noarch
  fakeroot-1.37.1-1.fc42.x86_64
  fakeroot-libs-1.37.1-1.fc42.x86_64
  filesystem-srpm-macros-3.18-47.fc42.noarch
  firacode-nerd-fonts-3.4.0-1.fc42.noarch
  fonts-srpm-macros-1:2.0.5-22.fc42.noarch
  forge-srpm-macros-0.4.0-2.fc42.noarch
  fpc-srpm-macros-1.3-14.fc42.noarch
  gcc-plugin-annobin-15.2.1-1.fc42.x86_64
  ghc-srpm-macros-1.9.2-2.fc42.noarch
  gnat-srpm-macros-6-7.fc42.noarch
  go-srpm-macros-3.8.0-1.fc42.noarch
  grubby-8.40-82.fc42.x86_64
  jetbrains-mono-fonts-2.304-8.fc42.noarch
  jetbrains-mono-fonts-all-2.304-8.fc42.noarch
  jetbrains-mono-nl-fonts-2.304-8.fc42.noarch
  kernel-srpm-macros-1.0-25.fc42.noarch
  kmod-zfs-2.3.4-1.fc42.x86_64
  kmodtool-1.1-13.fc42.noarch
  libnvpair3-2.3.4-1.fc42.x86_64
  libuutil3-2.3.4-1.fc42.x86_64
  libzfs6-2.3.4-1.fc42.x86_64
  libzpool6-2.3.4-1.fc42.x86_64
  lua-srpm-macros-1-15.fc42.noarch
  nerdfontssymbolsonly-nerd-fonts-3.4.0-1.fc42.noarch
  ocaml-srpm-macros-10-4.fc42.noarch
  openblas-srpm-macros-2-19.fc42.noarch
  package-notes-srpm-macros-0.5-13.fc42.noarch
  pcp-conf-6.3.7-8.fc42.x86_64
  pcp-libs-6.3.7-8.fc42.x86_64
  perl-srpm-macros-1-57.fc42.noarch
  pv-1.9.34-1.fc42.x86_64
  pyproject-srpm-macros-1.18.4-1.fc42.noarch
  python-srpm-macros-3.13-5.fc42.noarch
  python3-progressbar2-4.5.0-1.fc42.noarch
  python3-pyzfs-2.3.4-1.fc42.noarch
  python3-rpmautospec-0.8.1-11.fc42.noarch
  python3-rpmautospec-core-0.1.5-3.fc42.noarch
  python3-utils-3.9.0-2.fc42.noarch
  qt5-srpm-macros-5.15.17-1.fc42.noarch
  qt6-srpm-macros-6.9.2-1.fc42.noarch
  redhat-rpm-config-342-4.fc42.noarch
  rpm-build-4.20.1-1.fc42.x86_64
  rpmdevtools-9.6-9.fc42.noarch
  rsms-inter-fonts-4.1-1.fc42.noarch
  rust-srpm-macros-26.4-1.fc42.noarch
  starship-1.23.0-1.fc42.x86_64
  sysstat-12.7.7-1.fc42.x86_64
  systemd-rpm-macros-257.10-1.fc42.noarch
  tree-sitter-srpm-macros-0.1.0-8.fc42.noarch
  zfs-2.3.4-1.fc42.x86_64
  zig-srpm-macros-1-4.fc42.noarch
Added:
  NetworkManager-team-1:1.52.1-1.fc42.x86_64
  SDL2_image-2.8.8-1.fc42.x86_64
  SLOF-20220719-6.git6b6c16b4.fc42.noarch
  android-tools-1:35.0.2-5.fc42.x86_64
  appstream-compose-1.1.0-1.fc42.x86_64
  bcc-0.33.0-1.fc42.x86_64
  bcc-tools-0.33.0-1.fc42.x86_64
  boost-iostreams-1.83.0-12.fc42.x86_64
  bpftop-0.7.1-1.fc42.x86_64
  bpftrace-0.23.5-1.fc42.x86_64
  capstone-5.0.1-7.fc42.x86_64
  ccache-4.10.2-2.fc42.x86_64
  chkconfig-1.33-1.fc42.x86_64
  clang-libs-20.1.8-4.fc42.x86_64
  clang-resource-filesystem-20.1.8-4.fc42.x86_64
  clang19-libs-19.1.7-13.fc42.x86_64
  clang19-resource-filesystem-19.1.7-13.fc42.x86_64
  clevis-21-10.fc42.x86_64
  clevis-luks-21-10.fc42.x86_64
  clevis-pin-tpm2-0.5.3-9.fc42.x86_64
  cockpit-bridge-347-1.fc42.noarch
  cockpit-machines-341-1.fc42.noarch
  cockpit-networkmanager-347-1.fc42.noarch
  cockpit-ostree-1:216-1.fc42.noarch
  cockpit-podman-115-1.fc42.noarch
  cockpit-selinux-347-1.fc42.noarch
  cockpit-storaged-347-1.fc42.noarch
  cockpit-system-347-1.fc42.noarch
  code-1.105.1-1760482588.el8.x86_64
  compiler-rt-20.1.8-4.fc42.x86_64
  compiler-rt19-19.1.7-13.fc42.x86_64
  containerd.io-1.7.28-1.fc42.x86_64
  cowsql-1.15.9-1.fc42.x86_64
  daxctl-libs-82-1.fc42.x86_64
  device-mapper-multipath-0.10.0-5.fc42.x86_64
  device-mapper-multipath-libs-0.10.0-5.fc42.x86_64
  dhcpcd-10.1.0-2.fc42.x86_64
  docker-buildx-plugin-0.29.1-1.fc42.x86_64
  docker-ce-3:28.5.1-1.fc42.x86_64
  docker-ce-cli-1:28.5.1-1.fc42.x86_64
  docker-ce-rootless-extras-28.5.1-1.fc42.x86_64
  docker-compose-plugin-2.40.2-1.fc42.x86_64
  docker-model-plugin-0.1.44-1.fc42.x86_64
  dyninst-13.0.0-5.fc42.x86_64
  edk2-aarch64-20250523-16.fc42.noarch
  edk2-ovmf-20250523-16.fc42.noarch
  edk2-riscv64-20250523-16.fc42.noarch
  edk2-shell-aa64-20250523-16.fc42.noarch
  edk2-shell-x64-20250523-16.fc42.noarch
  flatpak-builder-1.4.4-3.fc42.x86_64
  gdisk-1.0.10-3.fc42.x86_64
  git-subtree-2.51.0-2.fc42.noarch
  git-svn-2.51.0-2.fc42.noarch
  glusterfs-11.2-1.fc42.x86_64
  glusterfs-cli-11.2-1.fc42.x86_64
  glusterfs-client-xlators-11.2-1.fc42.x86_64
  glusterfs-fuse-11.2-1.fc42.x86_64
  gnutls-utils-3.8.10-1.fc42.x86_64
  golang-github-opencontainers-umoci-0.4.7-0.2.fc41.x86_64
  gperftools-libs-2.15-5.fc42.x86_64
  gtk-vnc2-1.5.0-3.fc42.x86_64
  guestfs-tools-1.54.0-1.fc42.x86_64
  gvisor-tap-vsock-6:0.8.6-1.fc42.x86_64
  gvisor-tap-vsock-gvforwarder-6:0.8.6-1.fc42.x86_64
  gvnc-1.5.0-3.fc42.x86_64
  hexedit-1.6-8.fc42.x86_64
  hipcc-18-37.rocm6.3.1.fc42.x86_64
  hivex-libs-1.3.24-3.fc42.x86_64
  incus-6.15-1.fc42.x86_64
  incus-agent-6.15-1.fc42.x86_64
  incus-client-6.15-1.fc42.x86_64
  incus-selinux-6.15-1.fc42.noarch
  initscripts-10.26-3.fc42.x86_64
  initscripts-rename-device-10.26-3.fc42.x86_64
  iotop-c-1.30-1.fc42.x86_64
  iproute-tc-6.12.0-3.fc42.x86_64
  ipxe-roms-qemu-20240119-2.gitde8a0821.fc41.noarch
  iscsi-initiator-utils-6.2.1.11-0.git4b3e853.fc42.x86_64
  iscsi-initiator-utils-iscsiuio-6.2.1.11-0.git4b3e853.fc42.x86_64
  isns-utils-libs-0.103-2.fc42.x86_64
  jose-14-4.fc42.x86_64
  kcli-99.0.0.git.202510240548.236751f-0.fc42.x86_64
  libblkio-1.5.0-2.fc41.x86_64
  libblockdev-btrfs-3.3.1-2.fc42.x86_64
  libblockdev-lvm-3.3.1-2.fc42.x86_64
  libburn-1.5.6-6.fc42.x86_64
  libcacard-3:2.8.1-11.fc42.x86_64
  libcgroup-3.0-8.fc42.x86_64
  libconfig-1.7.3-11.fc42.x86_64
  libfdt-1.7.2-3.fc42.x86_64
  libgfapi0-11.2-1.fc42.x86_64
  libgfrpc0-11.2-1.fc42.x86_64
  libgfxdr0-11.2-1.fc42.x86_64
  libglusterfs0-11.2-1.fc42.x86_64
  libguestfs-1:1.56.2-1.fc42.x86_64
  libguestfs-appliance-1:1.56.2-1.fc42.x86_64
  libguestfs-xfs-1:1.56.2-1.fc42.x86_64
  libiscsi-1.20.0-4.fc42.x86_64
  libisoburn-1.5.6-7.fc42.x86_64
  libisofs-1.5.6-6.fc42.x86_64
  libldm-0.2.4-16.fc41.x86_64
  libluksmeta-9-24.fc42.x86_64
  libnbd-1.22.4-1.fc42.x86_64
  libnfs-6.0.2-6.fc42.x86_64
  libnl3-cli-3.11.0-3.fc42.x86_64
  libomp-20.1.8-4.fc42.x86_64
  libomp-devel-20.1.8-4.fc42.x86_64
  libomp19-19.1.7-13.fc42.x86_64
  libomp19-devel-19.1.7-13.fc42.x86_64
  libosinfo-1.12.0-2.fc42.x86_64
  libpanel-1.10.2-1.fc42.x86_64
  libphodav-3.0-11.fc42.x86_64
  libpmem-2.1.0-3.fc42.x86_64
  librados2-2:19.2.3-7.fc42.x86_64
  librbd1-2:19.2.3-7.fc42.x86_64
  librdmacm-55.0-1.fc42.x86_64
  libretls-3.8.1-5.fc42.x86_64
  libserf-1.3.10-9.fc42.x86_64
  libssh2-1.11.1-3.fc42.x86_64
  libsysprof-48.1-1.fc42.x86_64
  libteam-1.32-11.fc42.x86_64
  libtpms-0.10.1-1.fc42.x86_64
  libtraceevent-1.8.4-3.fc42.x86_64
  libtracefs-1.8.1-2.fc42.x86_64
  libvirt-11.0.0-4.fc42.x86_64
  libvirt-client-11.0.0-4.fc42.x86_64
  libvirt-client-qemu-11.0.0-4.fc42.x86_64
  libvirt-daemon-11.0.0-4.fc42.x86_64
  libvirt-daemon-common-11.0.0-4.fc42.x86_64
  libvirt-daemon-config-network-11.0.0-4.fc42.x86_64
  libvirt-daemon-config-nwfilter-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-ch-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-interface-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-libxl-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-lxc-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-network-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-nodedev-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-nwfilter-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-qemu-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-secret-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-core-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-disk-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-gluster-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-iscsi-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-iscsi-direct-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-logical-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-mpath-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-rbd-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-scsi-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-storage-zfs-11.0.0-4.fc42.x86_64
  libvirt-daemon-driver-vbox-11.0.0-4.fc42.x86_64
  libvirt-daemon-kvm-11.0.0-4.fc42.x86_64
  libvirt-daemon-lock-11.0.0-4.fc42.x86_64
  libvirt-daemon-log-11.0.0-4.fc42.x86_64
  libvirt-daemon-plugin-lockd-11.0.0-4.fc42.x86_64
  libvirt-daemon-proxy-11.0.0-4.fc42.x86_64
  libvirt-dbus-1.4.1-6.fc42.x86_64
  libvirt-glib-5.0.0-6.fc42.x86_64
  libvirt-libs-11.0.0-4.fc42.x86_64
  libvirt-nss-11.0.0-4.fc42.x86_64
  libvirt-ssh-proxy-11.0.0-4.fc42.x86_64
  libwsman1-2.7.2-12.fc42.x86_64
  libxdp-1.5.7-1.fc42.x86_64
  lldb-20.1.8-4.fc42.x86_64
  llvm19-filesystem-19.1.7-13.fc42.x86_64
  llvm19-libs-19.1.7-13.fc42.x86_64
  lsscsi-0.32-13.fc42.x86_64
  luksmeta-9-24.fc42.x86_64
  lxc-6.0.5-1.fc42.x86_64
  lxc-libs-6.0.5-1.fc42.x86_64
  lxc-templates-6.0.5-1.fc42.x86_64
  lxcfs-6.0.5-1.fc42.x86_64
  lzop-1.04-16.fc42.x86_64
  mdevctl-1.4.0-1.fc42.x86_64
  mingw-srvany-redistributable-1.1-10.fc42.noarch
  mkisofs-10:2024.03.21-5.fc42.x86_64
  nbdkit-1.42.8-1.fc42.x86_64
  nbdkit-basic-filters-1.42.8-1.fc42.x86_64
  nbdkit-basic-plugins-1.42.8-1.fc42.x86_64
  nbdkit-curl-plugin-1.42.8-1.fc42.x86_64
  nbdkit-nbd-plugin-1.42.8-1.fc42.x86_64
  nbdkit-python-plugin-1.42.8-1.fc42.x86_64
  nbdkit-selinux-1.42.8-1.fc42.noarch
  nbdkit-server-1.42.8-1.fc42.x86_64
  nbdkit-ssh-plugin-1.42.8-1.fc42.x86_64
  nbdkit-vddk-plugin-1.42.8-1.fc42.x86_64
  ndctl-libs-82-1.fc42.x86_64
  netcat-1.229-2.fc42.x86_64
  nicstat-1.95-25.fc42.x86_64
  nmap-ncat-4:7.92-5.fc42.x86_64
  numactl-2.0.19-2.fc42.x86_64
  numad-0.5-46.20150602git.fc42.x86_64
  openbios-1:20230126-5.gitaf97fd7.fc42.noarch
  opencl-filesystem-1.0-22.fc42.noarch
  osbuild-161-1.fc42.noarch
  osbuild-selinux-161-1.fc42.noarch
  osinfo-db-20250124-1.fc42.noarch
  osinfo-db-tools-1.12.0-2.fc42.x86_64
  p7zip-16.02-32.fc42.x86_64
  perl-Git-SVN-2.51.0-2.fc42.noarch
  perl-Memoize-1.16-519.fc42.noarch
  perl-YAML-1.31-5.fc42.noarch
  podman-bootc-v0.1.1~145~g829cfb4-1.20250805142308570730.main.22.829cfb4.fc42.x86_64
  podman-compose-1.5.0-1.1.fc42.noarch
  podman-machine-5:5.6.2-1.fc42.x86_64
  podman-tui-1.9.0-1.fc42.x86_64
  podmansh-5:5.6.2-1.fc42.x86_64
  powerline-fonts-2.8.4-5.fc42.noarch
  protobuf-3.19.6-11.fc42.x86_64
  python3-bcc-0.33.0-1.fc42.noarch
  python3-dasbus-1.7-9.fc42.noarch
  python3-dotenv-1.0.1-4.fc42.noarch
  python3-dotenv+cli-1.0.1-4.fc42.noarch
  python3-librepo-1.20.0-1.fc42.x86_64
  python3-libvirt-11.0.0-1.fc42.x86_64
  python3-libxml2-2.12.10-1.fc42.x86_64
  python3-lldb-20.1.8-4.fc42.x86_64
  python3-lxml-5.3.2-1.fc42.x86_64
  python3-osbuild-161-1.fc42.noarch
  python3-prettytable-3.10.0-7.fc42.noarch
  python3-wcwidth-0.2.13-6.fc42.noarch
  qatlib-24.09.0-6.fc42.x86_64
  qatlib-service-24.09.0-6.fc42.x86_64
  qatzip-libs-1.3.1-1.fc42.x86_64
  qemu-2:9.2.4-2.fc42.x86_64
  qemu-audio-alsa-2:9.2.4-2.fc42.x86_64
  qemu-audio-dbus-2:9.2.4-2.fc42.x86_64
  qemu-audio-jack-2:9.2.4-2.fc42.x86_64
  qemu-audio-oss-2:9.2.4-2.fc42.x86_64
  qemu-audio-pa-2:9.2.4-2.fc42.x86_64
  qemu-audio-pipewire-2:9.2.4-2.fc42.x86_64
  qemu-audio-sdl-2:9.2.4-2.fc42.x86_64
  qemu-audio-spice-2:9.2.4-2.fc42.x86_64
  qemu-block-blkio-2:9.2.4-2.fc42.x86_64
  qemu-block-curl-2:9.2.4-2.fc42.x86_64
  qemu-block-dmg-2:9.2.4-2.fc42.x86_64
  qemu-block-gluster-2:9.2.4-2.fc42.x86_64
  qemu-block-iscsi-2:9.2.4-2.fc42.x86_64
  qemu-block-nfs-2:9.2.4-2.fc42.x86_64
  qemu-block-rbd-2:9.2.4-2.fc42.x86_64
  qemu-block-ssh-2:9.2.4-2.fc42.x86_64
  qemu-char-baum-2:9.2.4-2.fc42.x86_64
  qemu-char-spice-2:9.2.4-2.fc42.x86_64
  qemu-common-2:9.2.4-2.fc42.x86_64
  qemu-device-display-qxl-2:9.2.4-2.fc42.x86_64
  qemu-device-display-vhost-user-gpu-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-ccw-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-gl-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-pci-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-pci-gl-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-pci-rutabaga-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-gpu-rutabaga-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-vga-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-vga-gl-2:9.2.4-2.fc42.x86_64
  qemu-device-display-virtio-vga-rutabaga-2:9.2.4-2.fc42.x86_64
  qemu-device-usb-host-2:9.2.4-2.fc42.x86_64
  qemu-device-usb-redirect-2:9.2.4-2.fc42.x86_64
  qemu-device-usb-smartcard-2:9.2.4-2.fc42.x86_64
  qemu-img-2:9.2.4-2.fc42.x86_64
  qemu-kvm-2:9.2.4-2.fc42.x86_64
  qemu-kvm-core-2:9.2.4-2.fc42.x86_64
  qemu-pr-helper-2:9.2.4-2.fc42.x86_64
  qemu-system-aarch64-2:9.2.4-2.fc42.x86_64
  qemu-system-aarch64-core-2:9.2.4-2.fc42.x86_64
  qemu-system-alpha-2:9.2.4-2.fc42.x86_64
  qemu-system-alpha-core-2:9.2.4-2.fc42.x86_64
  qemu-system-arm-2:9.2.4-2.fc42.x86_64
  qemu-system-arm-core-2:9.2.4-2.fc42.x86_64
  qemu-system-avr-2:9.2.4-2.fc42.x86_64
  qemu-system-avr-core-2:9.2.4-2.fc42.x86_64
  qemu-system-loongarch64-2:9.2.4-2.fc42.x86_64
  qemu-system-loongarch64-core-2:9.2.4-2.fc42.x86_64
  qemu-system-m68k-2:9.2.4-2.fc42.x86_64
  qemu-system-m68k-core-2:9.2.4-2.fc42.x86_64
  qemu-system-microblaze-2:9.2.4-2.fc42.x86_64
  qemu-system-microblaze-core-2:9.2.4-2.fc42.x86_64
  qemu-system-mips-2:9.2.4-2.fc42.x86_64
  qemu-system-mips-core-2:9.2.4-2.fc42.x86_64
  qemu-system-or1k-2:9.2.4-2.fc42.x86_64
  qemu-system-or1k-core-2:9.2.4-2.fc42.x86_64
  qemu-system-ppc-2:9.2.4-2.fc42.x86_64
  qemu-system-ppc-core-2:9.2.4-2.fc42.x86_64
  qemu-system-riscv-2:9.2.4-2.fc42.x86_64
  qemu-system-riscv-core-2:9.2.4-2.fc42.x86_64
  qemu-system-rx-2:9.2.4-2.fc42.x86_64
  qemu-system-rx-core-2:9.2.4-2.fc42.x86_64
  qemu-system-s390x-2:9.2.4-2.fc42.x86_64
  qemu-system-s390x-core-2:9.2.4-2.fc42.x86_64
  qemu-system-sh4-2:9.2.4-2.fc42.x86_64
  qemu-system-sh4-core-2:9.2.4-2.fc42.x86_64
  qemu-system-sparc-2:9.2.4-2.fc42.x86_64
  qemu-system-sparc-core-2:9.2.4-2.fc42.x86_64
  qemu-system-tricore-2:9.2.4-2.fc42.x86_64
  qemu-system-tricore-core-2:9.2.4-2.fc42.x86_64
  qemu-system-x86-2:9.2.4-2.fc42.x86_64
  qemu-system-x86-core-2:9.2.4-2.fc42.x86_64
  qemu-system-xtensa-2:9.2.4-2.fc42.x86_64
  qemu-system-xtensa-core-2:9.2.4-2.fc42.x86_64
  qemu-tools-2:9.2.4-2.fc42.x86_64
  qemu-ui-curses-2:9.2.4-2.fc42.x86_64
  qemu-ui-egl-headless-2:9.2.4-2.fc42.x86_64
  qemu-ui-gtk-2:9.2.4-2.fc42.x86_64
  qemu-ui-opengl-2:9.2.4-2.fc42.x86_64
  qemu-ui-sdl-2:9.2.4-2.fc42.x86_64
  qemu-ui-spice-app-2:9.2.4-2.fc42.x86_64
  qemu-ui-spice-core-2:9.2.4-2.fc42.x86_64
  qemu-user-2:9.2.4-2.fc42.x86_64
  qemu-user-binfmt-2:9.2.4-2.fc42.x86_64
  qemu-user-static-2:9.2.4-2.fc42.x86_64
  qemu-user-static-alpha-2:9.2.4-2.fc42.x86_64
  qemu-user-static-arm-2:9.2.4-2.fc42.x86_64
  qemu-user-static-hexagon-2:9.2.4-2.fc42.x86_64
  qemu-user-static-hppa-2:9.2.4-2.fc42.x86_64
  qemu-user-static-loongarch64-2:9.2.4-2.fc42.x86_64
  qemu-user-static-m68k-2:9.2.4-2.fc42.x86_64
  qemu-user-static-microblaze-2:9.2.4-2.fc42.x86_64
  qemu-user-static-mips-2:9.2.4-2.fc42.x86_64
  qemu-user-static-or1k-2:9.2.4-2.fc42.x86_64
  qemu-user-static-ppc-2:9.2.4-2.fc42.x86_64
  qemu-user-static-riscv-2:9.2.4-2.fc42.x86_64
  qemu-user-static-s390x-2:9.2.4-2.fc42.x86_64
  qemu-user-static-sh4-2:9.2.4-2.fc42.x86_64
  qemu-user-static-sparc-2:9.2.4-2.fc42.x86_64
  qemu-user-static-x86-2:9.2.4-2.fc42.x86_64
  qemu-user-static-xtensa-2:9.2.4-2.fc42.x86_64
  raft-0.22.1-3.fc42.x86_64
  rocm-clang-18-37.rocm6.3.1.fc42.x86_64
  rocm-clang-devel-18-37.rocm6.3.1.fc42.x86_64
  rocm-clang-libs-18-37.rocm6.3.1.fc42.x86_64
  rocm-clang-runtime-devel-18-37.rocm6.3.1.fc42.x86_64
  rocm-comgr-18-37.rocm6.3.1.fc42.x86_64
  rocm-device-libs-18-37.rocm6.3.1.fc42.x86_64
  rocm-hip-6.3.1-4.fc42.x86_64
  rocm-libc++-18-37.rocm6.3.1.fc42.x86_64
  rocm-libc++-devel-18-37.rocm6.3.1.fc42.x86_64
  rocm-lld-18-37.rocm6.3.1.fc42.x86_64
  rocm-llvm-18-37.rocm6.3.1.fc42.x86_64
  rocm-llvm-devel-18-37.rocm6.3.1.fc42.x86_64
  rocm-llvm-filesystem-18-37.rocm6.3.1.fc42.x86_64
  rocm-llvm-libs-18-37.rocm6.3.1.fc42.x86_64
  rocm-llvm-static-18-37.rocm6.3.1.fc42.x86_64
  rocm-opencl-6.3.1-4.fc42.x86_64
  rocm-runtime-6.3.1-4.fc42.x86_64
  rocm-smi-6.3.1-3.fc42.x86_64
  rutabaga-gfx-ffi-0.1.3-3.fc42.x86_64
  schily-libs-10:2024.03.21-5.fc42.x86_64
  scrub-2.6.1-10.fc42.x86_64
  seabios-bin-1.17.0-5.fc42.noarch
  seavgabios-bin-1.17.0-5.fc42.noarch
  setroubleshoot-plugins-3.3.15-2.fc42.noarch
  setroubleshoot-server-3.3.35-4.fc42.x86_64
  spice-glib-0.42-6.fc42.x86_64
  spice-gtk3-0.42-6.fc42.x86_64
  spice-server-0.15.1-8.fc42.x86_64
  subversion-1.14.5-14.fc42.x86_64
  subversion-libs-1.14.5-14.fc42.x86_64
  subversion-perl-1.14.5-14.fc42.x86_64
  supermin-5.3.5-4.fc42.x86_64
  swtpm-0.10.1-1.fc42.x86_64
  swtpm-libs-0.10.1-1.fc42.x86_64
  swtpm-selinux-0.10.1-1.fc42.noarch
  swtpm-tools-0.10.1-1.fc42.x86_64
  syslinux-6.04-0.31.fc42.x86_64
  syslinux-extlinux-6.04-0.31.fc42.x86_64
  syslinux-extlinux-nonlinux-6.04-0.31.fc42.noarch
  syslinux-nonlinux-6.04-0.31.fc42.noarch
  sysprof-48.1-1.fc42.x86_64
  sysprof-cli-48.1-1.fc42.x86_64
  systemtap-client-5.3-1.fc42.x86_64
  systemtap-runtime-5.3-1.fc42.x86_64
  teamd-1.32-11.fc42.x86_64
  thrift-0.20.0-3.fc41.x86_64
  tiptop-2.3.2-4.fc42.x86_64
  trace-cmd-3.3.1-2.fc42.x86_64
  ublue-os-libvirt-workarounds-1.1-1.fc42.noarch
  udica-0.2.8-7.fc42.noarch
  udisks2-btrfs-2.10.91-1.fc42.x86_64
  udisks2-iscsi-2.10.91-1.fc42.x86_64
  udisks2-lvm2-2.10.91-1.fc42.x86_64
  usbredir-0.15.0-1.fc42.x86_64
  utf8proc-2.10.0-1.fc42.x86_64
  virglrenderer-1.1.0-2.fc42.x86_64
  virt-install-5.0.0-2.fc42.noarch
  virt-manager-5.0.0-2.fc42.noarch
  virt-manager-common-5.0.0-2.fc42.noarch
  virt-v2v-1:2.8.1-1.fc42.x86_64
  virt-viewer-11.0-15.fc42.x86_64
  virtiofsd-1.13.2-1.fc42.x86_64
  xdelta-3.1.0-22.fc42.x86_64
  xen-libs-4.19.3-4.fc42.x86_64
  xen-licenses-4.19.3-4.fc42.x86_64
  xorriso-1.5.6-7.fc42.x86_64
  ydotool-1.0.4-6.fc42.x86_64
  zerofree-1.1.1-15.fc42.x86_64
  zfs-fuse-0.7.2.2-32.fc42.x86_64
  zlib-ng-2.2.5-2.fc42.x86_64
```

renner0e commented on build_files/shared/build-base.sh:
```diff
@@ -52,9 +48,6 @@ echo "::endgroup::"
 
-# Beta
-# /ctx/build_files/base/10-beta.sh
-
```
did you accidentally remove this?

inffy replied:
Seems it was there two times?

https://github.com/ublue-os/aurora/pull/1142/files/05a4f33aca9eb9b6143eabcf138786ccd51b9c60#diff-f1fb8cbd7835dfd324589b257054c7710b52ef641f344e82d41f3095d0672f3aL50

inffy commented:
Think we are  ready (finally)

renner0e reviewed (approved):
LESS GOOO
